Changelog.md

```diff
@@ -10,6 +10,7 @@ Breaking changes:
  * Type Checker: Exponentiation and shifts of literals by non-literals will always use ``uint256`` or ``int256`` as a type.
  * Type Checker: Disallow structs and arrays in memory or calldata if they contain nested mappings.
  * Type Checker: Disallow assignments to state variables that contain nested mappings.
+ * Remove visibility of Libraries in inherited contracts.
```

Comment:
`using A for B` only affects the contract it is mentioned in and not all derived contracts